lib/OOCEapps/Controller/Patron.pm

```diff
@@ -71,6 +71,7 @@ sub webhook {
         $c->log->debug('handle '.$data->{type});
         my $cust_id = $c->data->{data}{object}{customer}
             || $c->data->{data}{object}{source}{customer}
+            || $c->data->{data}{object}{sources}{data}[0]{customer}
                 or die ['Webhook Unhandled:'.$c->app->dumper($data)];
```

Comment:
allenfalls waere es sinnvoll hier nicht zu sterben sondern auch so einen error status zurueckzusenden

Reply:
we catch the `die` here further down and return an error state (with logging)